playground/module-resolution/__tests__/module-resolution.spec.ts

```diff
@@ -116,7 +134,7 @@ describe.runIf(!isBuild)('module resolution', async () => {
 		});
 	});
 
-	describe('user errors', () => {
+	describe.skipIf(isBuild)('dev user errors', () => {
```

Comment:
the test here is testing the vite error overlay which is indeed a dev only vite feature so this actually should be skipped for builds